src/components/Garden/ModalFlows/ModalFlowBase.js

```diff
@@ -71,6 +77,8 @@ function ModalFlowBase({
                   // TODO: Provide a better user experience than waiting on all transactions
                   await tx.wait()
 
+                  setAccountSetting('lastTxHash', account, chainId, tx.hash)
```

Comment:
Maybe that string 'lastTxHash' put on some Enum inside local-settings?

Or convert that file to typescript and use 'lastTxHash' on type. 
What yours think about @rperez89 and @tmoutinho ?

Reply:
IMO we should try to migrate this files to typescript, but I understand that @rperez89 has no experience on that. Perhaps after this is merged I can try do migrate this files, as an example how we should do it. WDYT?

Reply:
Its look a good idea @tmoutinho. I'll need change the confetti thing to TS too, so we will have 2 examples on that.